### PR TITLE
📝 Add links to other orgs in footer

### PR DESCRIPTION
**Pull request by SohamG**
Signed-off-by: Soham S Gumaste <sohamg2@gmail.com>

Let me know what other orgs need to be added.

Also we probably should break up the docusaurus config into smaller json/js files according to feature.

### CODE REVIEW

File: docusaurus.config.js [hunk 1]
```diff
@@ -12,6 +12,11 @@ const gitHubRepoUrl = `${gitHubOrgUrl}/${gitHubRepoName}`;
 const gitHubRepoDefaultBranch = "main";
 const gitHubPagesUrl = `https://${gitHubRepoName}`;
 
+const lugLink = "https://acm.cs.uic.edu";
```
Comment (clee231) [inline docusaurus.config.js:15]: This is the wrong URL.

File: docusaurus.config.js [hunk 2]
```diff
@@ -154,6 +159,23 @@ const config = {
             { label: "Contact us", to: "/contact" },
           ],
         },
+        {
+          title: "Other Organizations",
+          items: [
+            {
+              label: "LUG@UIC",
```
Comment (clee231) [inline docusaurus.config.js:166]: Technically, all these organizations are `@UIC`. We can probably omit that and just name of the link `LUG`.

**Review by clee231 — CHANGES_REQUESTED** [review]

File: docusaurus.config.js (hunk 2 above)
Comment (bmiddha) [reply docusaurus.config.js:166]: In that case the title could be updated to reflect that the links are related to UIC.

File: docusaurus.config.js (hunk 1 above)
Comment (bmiddha) [reply docusaurus.config.js:15]: good catch

File: docusaurus.config.js (hunk 2 above)
Comment (clee231) [reply docusaurus.config.js:166]: Do you think we would ever have a need to add other non-UIC orgs? I know on the old site, we did have a link back to something like acm.org... but maybe that would go elsewhere.

Comment (bmiddha) [reply docusaurus.config.js:166]: Maybe.
We can consider something like "External Links" or perhaps have different headings for UIC things and other things?

Comment (SohamG) [reply docusaurus.config.js]: Ive made these changes. Anything else before we can close this?

**Comment by bmiddha** [comment]
Looks like you're including changes from another PR.
Please cleanup the branch with just your changes.
Other than that, looks good!

**Comment by SohamG** [comment]
i hate git

**Review by bmiddha — APPROVED** [review]

File: docusaurus.config.js [hunk 3]
```diff
@@ -154,6 +157,23 @@ const config = {
...
+        {
+          title: "Other UIC Organizations",
+          items: [
+            {
+              label: "LUG",
+              href: lugLink,
+            },
+            {
+              label: "LOGICA",
+              href: wicsLink,
```
Comment (bmiddha) [inline docusaurus.config.js:169]: use `logicaLink` here.

**Review by bmiddha — CHANGES_REQUESTED** [review]

**Review by bmiddha — APPROVED** [review]

**Review by clee231 — APPROVED** [review]

**Pull request merged**